README.md

```diff
@@ -2,22 +2,28 @@
 
 # Foundry Virtual Tabletop - Dungeons & Dragons Fifth Edition Game System
 
-This game system for [Foundry Virtual Tabletop](http://foundryvtt.com) provides character sheet and game system 
+This game system for [Foundry Virtual Tabletop](http://foundryvtt.com) provides character sheet and game system
 support for the Fifth Edition of the world's most popular roleplaying game.
 
 This system provides character sheet support for Actors and Items, mechanical support for dice and rules necessary to
-play games of Fifth Edition, and compendium content for Monsters, Heroes, Items, Spells, Class Features, Monster 
+play games of Fifth Edition, and compendium content for Monsters, Heroes, Items, Spells, Class Features, Monster
 Features, and more!
 
+## Licenses
+
 This work includes material taken from the System Reference Document 5.1 ("SRD 5.1") by Wizards of the Coast LLC and available at https://dnd.wizards.com/resources/systems-reference-document. The SRD 5.1 is licensed under the Creative Commons Attribution 4.0 International License available at https://creativecommons.org/licenses/by/4.0/legalcode.
 
+This work includes material from the System Reference Document 5.2 (“SRD 5.2”) by Wizards of the Coast
```

Comment:
I totally missed that this file is hard-wrapped. Can we apply the same to the above line as well then?